__tests__/contentPreference.ts

```diff
@@ -103,32 +103,45 @@ describe('query userFollowers', () => {
       }),
     );
 
+    await saveFixtures(
+      con,
+      Feed,
+      usersFixture.map((item) => ({
+        id: `${item.id}-ufq`,
+        userId: `${item.id}-ufq`,
+      })),
+    );
+
     const now = new Date();
 
     await con.getRepository(ContentPreferenceUser).save([
       {
         userId: '2-ufq',
+        feedId: '2-ufq',
```

Comment:
How did this test not fail before 😢

Reply:
But maybe we should add 1 user - 3 feeds with same tags for safety

Reply:
Added in 2815f85c2e0863a391f38b3d0f34048a3c75ef8c